### ISSUE TITLE
Does gate way server war file from 0.4 require Tomcat to run?

### ISSUE THREAD

**Issue by iamgzy** [AbsaOSS/spline#544]
Happy new year Spline team ! 

## Background [Optional]
Tried to run spline 0.4 gate way server war but experienced following error
>java -jar /home/tempuser/spline.0.4/spline-rest-server-0.4.1.war -Dspline.database.connectionUrl=arangodb://localhost/spline

>no main manifest attribute, in /home/tempuser/spline.0.4/spline-rest-server-0.4.1.war  
I tried the war file from both maven repo and build it from source code, got the same error

## Question
In Spline 0.3.9 the war file is an executible standalone web application (Spring-boot app), but 0.4 war seems require tomcat container to deploy the war file? If so it seems not a big deal on testing environment but on production that means to add another dependency in the deployment process. 
So far docker is not an option.

**Comment by wajda** [comment]
Hi @iamgzy, happy new year too!
The executable Jar (not war) in 0.3 had an embedded Tomcat in it with the war pre-deployed. That was an easy-to-play-with solution, but it also had some issues and the lack of control on the embedded server. It in turn made it difficult to troubleshoot, especially on production. That's why we decided to replace the executable Jar with some more robust, flexible and production-ready solution.
So we went for the Docker-container - https://hub.docker.com/u/absaoss
